content/en/docs/release-notes/_index.md

```diff
@@ -9,6 +9,7 @@ no_list: true
 Here you will find a link to all release notes for each version release of
 cert-manager:
 
+- [`v1.3`](./release-notes-1.3/)
 - [`v1.1`](./release-notes-1.1/)
```

Comment:
We should have committed the 1.2 release notes into release-next rather than master during the last release, but we didn't. So we'll have to fix that in another PR.